core/test/unit/data/exporter/index_spec.js

```diff
@@ -49,7 +49,7 @@ describe('Exporter', function () {
             // Execute
             exporter.doExport().then(function (exportData) {
                 // No tables, less the number of excluded tables
-                var expectedCallCount = schemaTables.length - 4;
+                var expectedCallCount = schemaTables.length - exporter.EXCLUDED_TABLES.length;
```

Comment:
Just gotten rid of this magic number as needed to fix the test case itself anyway